### DISCUSSION TITLE
Not booting correctly

### DISCUSSION THREAD

**Discussion by patrick47113** [jomjol/AI-on-the-edge-device#1083] [General]
Hi,
Today I started with this project. I installed the package on my ESP32-CAM using the espressif Tool. I also configured it with a reference image and all the configs needed and then I integrated it into Home Assistant. Everything worked fine. After some hours the WLAN connection started to get unstable. And now I'm not able to connect to it anymore. I also erased and reflashed it, but the LED is blinking 5 times slowly and a ping to it's IP address results in some replys and many timeouts. I also tried to use another SD card, without success. Any ideas about the reason for that and how to solve it? Thanks!

regards, Patrick

**Comment by patrick47113** [comment]
I think, my question can be ignored. I just found out, that I distroyed the camera while I was trying to adjust the focus. Probably this is the cause, why the module has this malfunction.

**Reply by caco3** [reply]
Thats a pitty :(
Is the flat ribbon cable damaged?
Maybe try to remove the cammera and attach it again. Notice the black lever which must be opened for this!

Do you see something on the USB log?

**Comment by patrick47113** [comment]
Yes, I damaged the flat cable *GRRR*.
I'm quite new to this topic. So, unfortunatelly, I don't know, what you mean with USB log? Would you explain it to me?

**Comment by caco3** [comment]
If the cable is damaged, then that will be the reason.

To see the log, you can use putty or any other UART terminal. just connect the USB cable yo your PC and a virtual COM device should show up.

**Reply by patrick47113** [reply]
I see only some weired signs in the Putty session. And as long as I have an open Putty session, the device doesn't o anything. As soon as I close the Putty session, it is starting to blinking again and sometimes I get a ping reply to it's ip address.

**Reply by jomjol** [reply]
Then you putty is not set to the correct speed and type (monitor_speed = 115200, rts = 0, dtr = 0)

**Reply by patrick47113** [reply]
OK, I set the speed to 115200, but how do I configure RTS and DTR? I tried all possible settings in Putty (I'm using Windows). I tried to remove the camera and the SD card. If I connect the ESP32-CAM to USB, the red LED starts blinking. As soon as I start Putty, the blinking stops, but the Putty windows only shows some weired symbols. When I close Putty, the ESP32-CAM starts blinking again.

**Reply by caco3** [reply]
no need to set RTS/DTR!
Putty should have no impact on the startup as it only reads!
It will not start without sd-card or camera!

**Comment by caco3** [comment]
Is it still open for you?
I created a task to add some documentation to the wiki: https://github.com/jomjol/AI-on-the-edge-device/issues/1093

**Comment by patrick47113** [comment]
I don't know by now. My camera is broken, so I think it might be because of that. I ordered a new one, but it's not delivered so far. As soon as I have it I can test and tell you, if I still have the problem or if it's fixed.
